TODO.md

```diff
@@ -65,11 +65,10 @@
   - [x] Kappa
   - [x] Kappa Unbiased
   - [x] Kappa No Prevalence
-  - [ ] Aickin's alpha
```

Comment:
Is this line deleted mistakenly?

Reply:
> Is this line deleted mistakenly?

@sadrasabouri Thanks for your review ;-)
Yeah, I will update `TODO.md`